scoped_async_spawn/src/guard.rs

```diff
@@ -87,15 +87,19 @@ impl<'s> SpawnGuard<'s> {
 
 impl<'s> Drop for SpawnedTracker<'s> {
     fn drop(&mut self) {
-        let res = std::panic::catch_unwind(AssertUnwindSafe(|| {
-            self.0.drain(..).for_each(|ch| {
-                if let Some(ch) = ch.upgrade() {
-                    ch.as_ref().drop_future_now();
-                }
-            })
-        }));
-        if res.is_err() {
-            std::process::abort();
+        struct AbortBomb;
+        impl Drop for AbortBomb {
+            fn drop(&mut self) {
+                let _bomb = AbortBomb;
```

Comment:
Do we need this line?

Reply:
Not really, it should never execute as long as panicking while a panic is already in progress leads to an abort. So this is really to guard against that changing. If it doesn't cause an abort we will instead call this nested abort bomb's drop implementation and panic again.